main.py

```diff
@@ -1,7 +1,88 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
+list_of_genres = [
+    "Western", "Action", "Dramma"
+]
+
+list_of_actors = [
+    {
+        "first_name": "George",
+        "last_name": "Klooney"
+    },
+    {
+        "first_name": "Kianu",
+        "last_name": "Reaves"
+    },
+    {
+        "first_name": "Scarlett",
+        "last_name": "Keegan"
+    },
+    {
+        "first_name": "Will",
+        "last_name": "Smith"
+    },
+    {
+        "first_name": "Jaden",
+        "last_name": "Smith"
+    },
+    {
+        "first_name": "Scarlett",
+        "last_name": "Johansson"
+    },
+
+]
+
 
 def main() -> QuerySet:
-    pass
+
+    for genre in list_of_genres:
+        Genre.objects.create(
+            name=f"{genre}"
+        )
+
+    for actor in list_of_actors:
+        Actor.objects.create(
+            first_name=actor["first_name"],
+            last_name=actor["last_name"]
+        )
```

Comment:
You could use bulk_create method to... umm... create objects in bulk :) That will increase your query performance greatly